tests/integration/tmp/.gitignore

```diff
@@ -0,0 +1 @@
+*.db
```

Comment:
I considered it but I cannot do that because the file has to be created in an init method, and `t.TmpDir` is tied to a specific test function.

The framework currently doesn't seem to have a way to create a temp folder for the entire test.

Reply:
Setup is run in the parent test of `Run` so is safe to use test scoped functions like `t.TmpDir`, see https://github.com/dapr/dapr/blob/cfbc8182e4c629038b0b2a0cc43dd52bd1df2684/tests/integration/integration.go#L41

The directory will be persisted into `Run`.

Reply:
Done